package.json

```diff
@@ -1,176 +1,103 @@
 {
-  "name": "react-apollo",
-  "version": "2.5.6",
-  "author": "opensource@apollographql.com",
+  "name": "apollo-react-monorepo",
+  "version": "3.0.0",
   "private": true,
-  "description": "React data container for Apollo Client",
-  "keywords": [
-    "apollo",
-    "graphql",
-    "react"
-  ],
+  "author": "opensource@apollographql.com",
   "license": "MIT",
-  "main": "lib/react-apollo.cjs.js",
-  "module": "lib/react-apollo.esm.js",
-  "react-native": {
-    "react-dom/server": false
-  },
-  "typings": "lib/index.d.ts",
```

Comment:
was this supposed to be removed? I am trying this on react native and it doesnt work

Reply:
Thanks @zackify - no this wasn't supposed to be removed. I'll get it added back in.